plugins/fabrik_element/fileupload/fileupload.php

```diff
@@ -2199,7 +2191,30 @@ public function render($data, $repeatCounter = 0)
 
 		$allRenders = implode('<br/>', $allRenders);
 		$allRenders .= ($allRenders == '') ? '' : '<br/>';
-		$str[] = $allRenders . '<input class="fabrikinput" name="' . $name . '" type="file" id="' . $id . '" />' . "\n";
+		$capture = "";
+		switch ($device_capture)
+		{
+			case 1:
+				$capture = ' capture="camera"';
+			case 2:
+				$capture = ' accept="image/*"' . $capture;
+				break;
+			case 3:
+				$capture = ' capture="microphone"';
+			case 4:
+				$capture = ' accept="audio/*"' . $capture;
+				break;
+			case 5:
+				$capture = ' capture="camcorder"';
+			case 6:
+				$capture = ' accept="video/*"' . $capture;
+				break;
+			default:
+				$capture = implode(",.",$this->_getAllowedExtension());
+				$capture = $capture ? ' accept=".' . $capture . '"' : '';
```

Comment:
small point - it would be better to add a "break;" here - incase other case statements get added underneath it